src/e2e/actor/character.e2e.test.js

```diff
@@ -5,7 +5,21 @@ export const options = {
   displayName: 'The Character Sheet'
 };
 
-export default ({before, beforeEach, after, describe, it, expect, ...context}) => {
+// TODO: Write Monster roll tests
```

Comment:
This file is meant for tests for the Character actor type. We'll want a separate file for Monsters.